pandas/tests/groupby/test_function.py

```diff
@@ -459,35 +460,35 @@ def test_groupby_cumprod():
     tm.assert_series_equal(actual, expected)
 
 
-def test_ops_general():
-    ops = [('mean', np.mean),
-           ('median', np.median),
-           ('std', np.std),
-           ('var', np.var),
-           ('sum', np.sum),
-           ('prod', np.prod),
-           ('min', np.min),
-           ('max', np.max),
-           ('first', lambda x: x.iloc[0]),
-           ('last', lambda x: x.iloc[-1]),
-           ('count', np.size), ]
+def scipy_sem(*args, **kwargs):
     try:
         from scipy.stats import sem
+        return sem(*args, ddof=1, **kwargs)
     except ImportError:
-        pass
-    else:
-        ops.append(('sem', sem))
+        pytest.skip("No Scipy installed")
```

Comment:
Can you use `skip_if_no_scipy` from pandas.util._test_decorators instead?

Reply:
fixed